src/libraries/Microsoft.PowerFx.Interpreter/Values/LambdaFormulaValue.cs

```diff
@@ -15,12 +15,23 @@ internal class LambdaFormulaValue : FormulaValue
     {
         private readonly IntermediateNode _tree;
 
+        internal EvalVisitor Runner { get; }
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

Can these be private readonly? 
(especially if you add the helper we discussed offline)